planner/core/planbuilder.go

```diff
@@ -448,6 +448,12 @@ type PlanBuilder struct {
 	buildingViewStack set.StringSet
 	// renamingViewName is the name of the view which is being renamed.
 	renamingViewName string
+	// CreateView needs this information to check whether exists nested view.
+	underlyingViewNames set.StringSet
```

Comment:
This variable doesn't belong to https://github.com/pingcap/tidb/pull/19709, so please remove it.

Reply:
Done.